__tests__/e2e/scripts/setup_integration.sh

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-set -e
```

Comment:
Any reason for removing this?

Reply:
Because I no longer want this to fail on errors, I want it to continue running on errors, so I can grab the logs from the containers.

Reply:
Gotcha 👍🏽